react-carousel-component/my-app/src/Indicators.jsx

```diff
@@ -0,0 +1,31 @@
+import { FaRegCircle, FaCircle } from 'react-icons/fa';
+
+type props = {
+  count: Number,
+  currentIndex: Number,
+  onClick: Function,
```

Comment:
I recommend you never use the `Function` type. It refers to _any_ function so it does not provide much in the way of type checking.

You should instead declare a function type that takes the parameters you need and returns the type you need. Looking at your code, you might declare `onClick` as:
```
onClick: (i: number) => void
```